src/group_theory/order_of_element.lean

```diff
@@ -329,6 +334,19 @@ end cancel_monoid
 section group
 variables [group G] [add_group A] {x a} {i : ℤ}
 
+/-- Inverses of elements of finite order have finite order. -/
+@[to_additive "Inverses of elements of finite additive order have finite additive order."]
+lemma is_of_fin_order_inv (x : G) : is_of_fin_order x → is_of_fin_order x⁻¹ :=
+λ hx, (is_of_fin_order_iff_pow_eq_one _).mpr $ begin
+  rcases (is_of_fin_order_iff_pow_eq_one x).mp hx with ⟨n, npos, hn⟩,
+  refine ⟨n, npos, by simp_rw [inv_pow, hn, one_inv]⟩,
+end
+
+/-- Inverses of elements of finite order have finite order. -/
+@[to_additive "Inverses of elements of finite additive order have finite additive order."]
+lemma is_of_fin_order_inv_iff {x : G} : is_of_fin_order x ↔ is_of_fin_order x⁻¹ :=
```

Comment:
Please swap the two sides of the `↔`, and make this a `simp` lemma.

Reply:
Done!